pylint/extensions/docparams.py

```diff
@@ -83,6 +77,22 @@ class DocstringParameterChecker(BaseChecker):
                   # we can't use the same old_name for two different warnings
                   # {'old_names': [('W9009', 'missing-yields-doc')]},
                  ),
+        'W9015': ('"%s" missing in parameter documentation',
+                  'missing-param-doc',
+                  'Please add parameter declarations for all parameters.',
+                  {'old_names': [('W9003', 'missing-param-doc')]}),
```

Comment:
We should leave these two as is (the missing-param-doc, with the old codes W9003 and W9004), and add old-names only to the new ones.